vrp-pragmatic/src/format/solution/writer.rs

```diff
@@ -50,6 +53,10 @@ impl<W: Write> PragmaticSolution<W> for (&Solution, f64) {
     fn write_geo_json(&self, problem: &Problem, writer: &mut BufWriter<W>) -> Result<(), String> {
         write_geo_json(problem, self.0, writer)
     }
+
+    fn write_pragmatic_and_geo_json(&self, problem: &Problem, writer: &mut BufWriter<W>) -> Result<(), String> {
+        write_pragmatic_and_geo_json(problem, self.0, None, writer)
+    }
```

Comment:
I'm rethinking also this approach with three methods. Maybe it is better to have one single method which accepts additionally an enumeration which specifies what to serialize